### PR TITLE
Require certain tests to pass for authz-v2 branch

### PR DESCRIPTION
**Pull request by pitlv2109**
Require same required tests as **release-1.1**'s to pass in order to merge PRs to **authz-v2**.

### CODE REVIEW

**Comment by pitlv2109** [comment]
@yangminzhu This is the PR mentioned in https://github.com/istio/test-infra/pull/1168.
/cc @utka Let us know if we need to do anything else.

File: prow/config.yaml [hunk 1]
```diff
@@ -105,6 +105,17 @@ branch-protection:
                 - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2"
                 - "ci/circleci: e2e-pilot-cloudfoundry-v1alpha3-v2"
                 - "merges-blocked-needs-admin"
+            authz-v2:
+              protect: true
```
Comment (yangminzhu) [inline prow/config.yaml]: We don't need to make it protect, I think only admin is allowed to merge if it's protected.

Comment (pitlv2109) [reply prow/config.yaml]: Deleted. Thanks Yangmin!

**Pull request merged**